scripts/test/corelli/calibration/test_utils.py

```diff
@@ -10,14 +10,34 @@
 from os import path
 import unittest
 
-from corelli.calibration.utils import (apply_calibration, bank_numbers, calculate_tube_calibration, load_banks,
+from corelli.calibration.utils import (apply_calibration, bank_numbers, calibrate_tube, load_banks, calculate_peak_y_table,
                                        wire_positions)
 from mantid import AnalysisDataService, config
-from mantid.simpleapi import DeleteWorkspaces, LoadEmptyInstrument
+from mantid.simpleapi import (CreateEmptyTableWorkspace, DeleteWorkspaces, GroupWorkspaces, LoadEmptyInstrument,
+                              LoadNexusProcessed)
 
 
 class TestUtils(unittest.TestCase):
 
+    @classmethod
+    def setUpClass(cls) -> None:
+        r"""
+        Load the tests cases for calibrate_bank, consisting of data for only one bank
+        CORELLI_124023_bank10, tube 13 has shadows at pixel numbers quite different from the rest
+        """
+        config.appendDataSearchSubDir('CORELLI/calibration')
+        for directory in config.getDataSearchDirs():
+            if 'UnitTest' in directory:
+                data_dir = path.join(directory, 'CORELLI', 'calibration')
```

Comment:
Just bikeshedding, I think `pathlib` is preferred these days on Python 3.

Reply:
I guess so, but `os.path` does the job and I'm too lazy :smile:

Reply:
Yeah, going from Python 2 to Python 3 was painful enough. No worries, it's just me bikeshedding.